Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM openjdk:11-jdk-slim-buster AS build
+
+ARG RSK_RELEASE
+ENV RSK_VERSION $RSK_RELEASE
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update -y && \
+    apt-get install -y git curl gnupg
+
+RUN useradd -ms /bin/bash rsk
+USER rsk
+
+WORKDIR /home/rsk
+COPY --chown=rsk:rsk . ./
+
+RUN gpg --keyserver https://secchannel.rsk.co/SUPPORT.asc --recv-keys 1DC9157991323D23FD37BAA7A6DBEAC640C5A14B && \
+    gpg --verify --output SHA256SUMS SHA256SUMS.asc && \
+    sha256sum --check SHA256SUMS && \
+    ./configure.sh && \
```

Comment:
Not sure if as part of the build command you will also need to add the `DONT-COMMIT-settings.gradle` file in order to build, as in [here](https://github.com/rootstock/fed-dockerized-test/blob/3d83774d6c8b9340fdc8013c256b3b0474c0a10c/powpeg-node/Dockerfile#L47).

Reply:
I see that makes sense, thanks